client/db/src/bench.rs

```diff
@@ -254,15 +263,21 @@ impl<B: BlockT> StateBackend<HashFor<B>> for BenchmarkingState<B> {
 	{
 		if let Some(db) = self.db.take() {
 			let mut db_transaction = DBTransaction::new();
+			let mut bytes = 0;
 
 			for (key, (val, rc)) in transaction.drain() {
 				if rc > 0 {
 					db_transaction.put(0, &key, &val);
 				} else if rc < 0 {
 					db_transaction.delete(0, &key);
 				}
+				bytes += key.len() as u64;
 			}
 			db.write(db_transaction).map_err(|_| String::from("Error committing transaction"))?;
+			let _ = self.usage_info.try_borrow_mut().map(|mut usage_info| {
+				usage_info.writes.ops += 1;
+				usage_info.writes.bytes += bytes;
```

Comment:
Inaccurate tracking of writes hear, would appreciate some feedback on how to track those properly